cli/cli_app.go

```diff
@@ -43,15 +57,12 @@ VERSION:
 AUTHOR(S):
    {{range .Authors}}{{.}}{{end}}
    {{end}}
-`
+`, CMD_ACQUIRE_LOCK, CMD_RELEASE_LOCK, CMD_SPIN_UP, CMD_TEAR_DOWN)
```

Comment:
Depends how many we have, but for now, it's a nice way to make the purpose of those vars crystal clears.